openassessment/xblock/resolve_dates.py

```diff
@@ -24,6 +24,18 @@ class DateValidationError(Exception):
 DISTANT_FUTURE = dt.datetime(dt.MAXYEAR, 1, 1, tzinfo=pytz.utc)
 
 
+def get_current_time_zone(open_assessment_block):
```

Comment:
👍 Agreed. I'd even like to a see a property that gets the block's `user_service`, so the calls could then simply be `get_current_time_zone(self.user_service)`